lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -53,8 +53,7 @@ def add_copy_resources_script_phase(native_target)
             test_type = target.test_type_for_product_type(native_target.symbol_type)
             script_path = "${PODS_ROOT}/#{target.copy_resources_script_path_for_test_type(test_type).relative_path_from(target.sandbox.root)}"
             resource_paths = target.all_dependent_targets.flat_map do |dependent_target|
-              include_test_spec_paths = dependent_target == target
-              dependent_target.resource_paths(include_test_spec_paths)
+              dependent_target.resource_paths.values.flatten
```

Comment:
uh this might be wrong......will check again and see if a test covers this.

Reply:
nope this is correct. I verified this is covered by a test case already by removing the code on `master` and seeing a few tests fail.

Reply:
and nope again it is wrong. I mistakenly broke the test in this PR to give a false positive.